YouTube/scripts/services.js

```diff
@@ -0,0 +1,25 @@
+function Services(api, { statisticsURL, searchURL, nextURL }) {
```

Comment:
Please change the function name "Services" to another name which follows camelCase Notation

Reply:
named it as services